README.md

```diff
@@ -130,6 +159,15 @@ Just take your usual npm commands and replace the word `npm` with `dry`.
 | npm publish | dry publish |
 | npm x y z   | dry x y z   |
 
+#### dry commands : additionam parameters
+
+The dry command accepts some additional parameters
+
+--dry-packager `packager_cli_name` : specify the nodejs packager cli to proxy (default : `npm`)
```

Comment:
I think it is a good idea to allow users to use alternatives to npm (i.e. yarn).
But I think this PR should stay focused on dependency management.
Let's add this feature in a new PR.

Reply:
Ok, done